clients/html/src/app/config/client_configuration.ts

```diff
@@ -26,7 +26,6 @@ export interface ClientPreferences {
   tiered_package_types: Array<PackageTypes>;
   relationship_quote_calculator: QuoteCalculatorConstructor<RelationshipContributionModel>;
   tiered_quote_calculator: QuoteCalculatorConstructor<TieredContributionModel>;
-  default_state: string;
```

Comment:
This is causing errors in Angular as it is not declared [here](https://github.com/ideacrew/quoting_tool/pull/26/files#diff-2e3fbb903fc3bc2c7a139b3e9d162c5b07c495927e9ad6161d499b727419ed90R87).

Reply:
yes - this needs to be removed.